setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name = "mandrel",
-    version = "0.2.0",
+    version = "0.2.1",
```

Comment:
Please don't include this in the PR; the version bump and release gets determined upon or following merge (and in this case will likely result in a minor increment, since we aren't at 1.x yet and this is a significant public interface change).